src/contents/en-gb/doc_dotnet.md

```diff
@@ -0,0 +1,265 @@
+# SAFE .Net Desktop App Tutorial
+
+In this tutorial, we will create a [.Net Framework](https://docs.microsoft.com/en-us/dotnet/framework/) console application. The .NET Framework is a development platform for building apps for web, Windows, Windows Phone, Windows Server, and Microsoft Azure using C#, Visual Basic and F#. This tutorial can also be used for developing [.Net Core](https://github.com/dotnet/core) apps.
+
+We will use the [MaidSafe.SafeApp](https://www.nuget.org/packages/MaidSafe.SafeApp/) NuGet package which exposes the `safe_app` API to connect and interact with the SAFE Network.
+
+Download demo code for this tutorial from [here](https://github.com/maidsafe/safe-getting-started-dotnet/SafeDesktopExample).
+
+## Pre-requisites
+
+Before we start working on our first SAFE app, make sure you have the following tools installed to be able to follow this tutorial.
+
+- **Visual Studio**: You need visual studio to compile and run your app. Follow the instructions [here](https://visualstudio.microsoft.com/) to download and install the visual studio for your Operating System. .Net Framework and .Net Core SDKs are installed automatically while installing visual studio.
+

```

Comment:
remember to add in any basic knowledge (like the Xamarin page)

Reply:
not needed. This one is simply